packages/xstate-react/src/useService.ts

```diff
@@ -33,11 +9,18 @@ export function useService<
 >(
   service: Interpreter<TContext, any, TEvent, TTypestate>
 ): [State<TContext, TEvent, any, TTypestate>, PayloadSender<TEvent>] {
-  const serviceActor = useMemo(() => fromService(service), [service]);
+  if (process.env.NODE_ENV !== 'production' && !('machine' in service)) {
+    throw new Error(
+      `Attempted to use an actor-like object instead of a service in the useService() hook. Please use the useActor() hook instead.`
```

Comment:
Please change this:
https://github.com/davidkpiano/xstate/blob/fc95a08a4c8b192a1cf947be640de7e79c540e68/packages/xstate-react/src/useActor.ts#L47
to
```js
const subscription = actorRef.subscribe(emitted => setCurrent(emitted)); 
```
Otherwise we'll end up with this problem again: https://github.com/davidkpiano/xstate/pull/1594/files